shell/platform/embedder/tests/embedder_unittests_proctable.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+// Verifies that the proc table is fully populated.
+TEST(EmbedderProcTable, AllPointersProvided) {
+  FlutterEngineProcTable procs = {};
+  procs.struct_size = sizeof(FlutterEngineProcTable);
+  FlutterEngineGetProcAddresses(&procs);
+
+  void (**proc)() = reinterpret_cast<void (**)()>(&procs.create_aot_data);
+  const uint64_t end_address =
+      reinterpret_cast<uint64_t>(&procs) + procs.struct_size;
+  while (reinterpret_cast<uint64_t>(proc) < end_address) {
```

Comment:
This will fail on 32-bit hosts. Let's ask the compiler for the size of the pointer.

Reply:
Fixed to use `uintptr_t`